Marlin/src/inc/SanityCheck.h

```diff
@@ -1395,8 +1395,8 @@ static_assert(NUM_SERVOS <= NUM_SERVO_PLUGS, "NUM_SERVOS (or some servo index) i
       #error "BIQU MicroProbe requires a PROBE_ENABLE_PIN."
     #endif
 
-    #if ENABLED(FT_MOTION) && DISABLED(ENDSTOP_INTERRUPTS_FEATURE)
-      #error "BIQU Microprobe requires ENDSTOP_INTERRUPTS_FEATURE with FT_MOTION."
+    #if ENABLED(FT_MOTION) && DISABLED(ENDSTOP_INTERRUPTS_FEATURE) && !(Z_CURRENT_HOME < Z_CURRENT)
+      #error "BIQU Microprobe requires ENDSTOP_INTERRUPTS_FEATURE or Z_CURRENT_HOME lower than Z_CURRENT with FT_MOTION."
```

Comment:
We should be fixing the actual issue instead of adding more sanity checks/workarounds.